RampUpAuto/src/test/java/seleautomation/com/RampUpAuto/GitHubSighInTest.java

```diff
@@ -0,0 +1,145 @@
+package com.Github;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.testng.annotations.*;
+import org.testng.Assert;
+
+
+class GitHubSignInPage {
+
+    WebDriver driver;
+
+    GitHubSignInPage(WebDriver driver) {
+
+        this.driver = driver;
+
+    }
+
+    public void signIn(String username, String password) {
```

Comment:
So it is good practice that you divided your code into separate functions , but I would like if you could apply the principle of POM and separate the testing class from the UI logic 
For example , you can create new class let us call it `signInPage` and put this function `signIn `inside of it , also you can add these functions in it `isSignInSuccessful` and `isErrorMessageDisplayed` in the  `signInPage` class